website/source/docs/job-specification/reschedule.html.md

```diff
@@ -59,6 +61,25 @@ job "docs" {
    number of reschedule happen within it. If more than `attempts` number of
    failures happen with this interval, Nomad will not reschedule any more.
 
+- `delay` `(string: <varies>)` - Specifies the duration to wait before attempting
+  to reschedule a failed task. This is specified using a label suffix like "30s" or "1h".
+
+- `delay_function` `(string: <varies>)` - Specifies the function according to which
```

Comment:
"Specifies the function according to which the initial delay specified in `delay` changes." ->

"Specifies the function that is used to calculate subsequent reschedule delays. The initial delay is specified by the `delay` parameter"

Reply:
done